<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>77</y>
      <width>371</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>                                                  7 LED Berpola</string>
    </property>
    <widget class="QPushButton" name="pushButton_Exit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>exit</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_on">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>START</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>play.png</normaloff>play.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_off">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>stop.png</normaloff>stop.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_OpenSerial">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Serial</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>508</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
